src/confluent_kafka/src/Admin.c

```diff
@@ -481,6 +482,19 @@ static PyObject *Admin_create_topics (Handle *self, PyObject *args,
                                 goto err;
                         }
                 }
+                if(!newt->replica_assignment){
+                        if(newt->replication_factor!=-1 && newt->num_partitions!=-1){
```

Comment:
We're not strictly enforcing code style in this repo, but try to maintain the same code style as for the other code, which has much more spaces around parens, operators, etc, to improve readability.

Reply:
I have made the suggested changes